index.py

```diff
@@ -2,38 +2,66 @@
 import asciiplotter as ascp
 from browser import document, html
 print("Hello! https://github.com/tusharhero")
+
+def clear(evt):
+    document["output"].value = ""
+    document["error"].html = "$ I ate the graph"
+
+def check():
+    eqt = document["text-src"].value
+    size = document["size"].value
+    scaling = document["scaling"].value
+
+    i = int(size)
+    scale = int(scaling)
+
+    if (i > 250):
+        return False
+    else:
+        return [eqt, i, scale]
+
 def getequation(evt):
-    equation = document["text-src"].value
-    try:
-        size = int(document["size"].value)
-        scaling = float(document["scaling"].value)
-        graph = asc.getstring(ascp.plot(equation, img=asc.createblankimg(size, size), s=scaling))
+    # equation = document["text-src"].value
+    # try:
+    #     size = int(document["size"].value)
+    #     scaling = float(document["scaling"].value)
+    #     graph = asc.getstring(ascp.plot(equation, img=asc.createblankimg(size, size), s=scaling))
+    #     document["error"].html = "$ checking"
+    #     document["box"].width = size
+    #     document["output"].value = graph
+    #     document["error"].html = "$ done!"
+    # except Exception as e:
+    #     document["error"].html = "$ " + str(e)
+    checkData = check()
+
+    if (checkData != False):
+        eqt = checkData[0]
+        size = checkData[1]
+        scale = checkData[2]
+
+        graph = asc.getstring(ascp.plot(eqt, img=asc.createblankimg(size, size), s=scale))
         document["error"].html = "$ checking"
         document["box"].width = size
         document["output"].value = graph
         document["error"].html = "$ done!"
-    except Exception as e:
```

Comment:
why did you get rid of this? It is used to display any python errors directly in the webpage.